### ISSUE TITLE
New project creation command fails 

### ISSUE THREAD

**Issue by sjimenez77** [oasp/oasp4j#546]
The following command that appears [here](https://github.com/devonfw/devon/wiki/getting-started-creating-new-devonfw-application#step-3-create-the-new-application) in the Devon Guide and [here](https://github.com/oasp/oasp4j/wiki/tutorial-newapp#from-command-line) in the OASP4J Wiki in order to create a new application fails:

```bash
mvn -DarchetypeVersion=2.2.0 -DarchetypeGroupId=io.oasp.java.templates -DarchetypeArtifactId=oasp4j-template-server archetype:generate -DgroupId=io.oasp.application -DartifactId=sampleapp -Dversion=0.1-SNAPSHOT -Dpackage=io.oasp.application.sampleapp
```

**OUTPUT:**

```bash
[ERROR] Parser Exception: archetype-resources/core/src/test/java/general/common/TestUt 
[ERROR] org.apache.velocity.runtime.parser.ParseException: Encountered ".." at line 28 
Was expecting one of: 
    <RPAREN> ... 
    <WHITESPACE> ... 
    <WHITESPACE> ...
    ...
```

**Comment by kiran-vadla** [comment]
@antgauti and @sjimenez77 , fix has to be ideally pushed onto develop-2.3.0 branch and not develop branch. 

Thanks,
Kiran.

**Comment by antgauti** [comment]
@kiran-vadla , please tag this issue as a [Bug](https://github.com/oasp/oasp4j/labels/bug) because it's a blocking defect in the app generator.

**Comment by amarinso** [comment]
This seems to be addressed in 2.3.0

But the wiki on OASP4j seems not to have been updated from oasp-forge... I will open a separate issue
